packages/gdrcopy.spec

```diff
@@ -81,12 +79,13 @@ Group: System Environment/Libraries
 Requires: %{name} = %{version}-%{_release}%{?dist}
 BuildArch: noarch
 
+%if %{BUILD_KMOD} == 0
```

Comment:
is this an issue? As long as it builds in a separate package we're good?

Reply:
It saves a point of failure in the build process. Since we don't use it, we can just skip it and not spend effort on it.

Reply:
Hm yes but it quite a change and we have to track if for every new version so it's not without effort.

Maybe we should try to get this upstream? Open a PR there and see if we can get it in ?

Reply:
I rolled back this change. We follow upstream and the DKMS stuff will be always built.